doc/rados/operations/monitoring.rst

```diff
@@ -178,37 +178,35 @@ to a healthy state:
     2017-07-25 10:11:13.535493 mon.a mon.0 172.21.9.34:6789/0 110 : cluster [INF] Health check cleared: PG_DEGRADED (was: Degraded data redundancy: 2 pgs unclean, 2 pgs degraded, 2 pgs undersized)
     2017-07-25 10:11:13.535577 mon.a mon.0 172.21.9.34:6789/0 111 : cluster [INF] Cluster is now healthy
 
-
 Network Performance Checks
 --------------------------
 
-Ceph OSDs send heartbeat ping messages amongst themselves to monitor daemon availability.  We
-also use the response times to monitor network performance.
-While it is possible that a busy OSD could delay a ping response, we can assume
-that if a network switch fails multiple delays will be detected between distinct pairs of OSDs.
+Ceph OSDs send heartbeat ping messages to each other in order to  monitor daemon availability 
+and network performance. If a single delayed ping response is detected, this might indicate 
+nothing more than a busy OSD. But if multiple delays between distinct pairs of OSDs are 
+detected, this might indicate a failed network switch.
 
-By default we will warn about ping times which exceed 1 second (1000 milliseconds).
+By default, a ping time that exceeds 1 second (1000 milliseconds) raises a health check. For example:
```

Comment:
s/ping/heartbeat/
/check/warning state.

Reply:
@anthonyeleven - As far as I can tell, _health warning state_ isn't used in the documentation. I've been using _health check_ simply because of the page named "Health checks". But for all I know, there might be a significant difference in meaning between _health warning state_ and _health check_ that is relevant here and that could usefully guide future work on the documentation. I'll gladly accept input on the matter.

Reply:
My sense here is that a health _check_ is a specific thing that is tested, or an instance of executing it.  `HEALTH_WARN` / `HEALTH_ERR` to me are states, flags, status, conditions as a result of a check.

Reply:
@anthonyeleven - The "Health checks" page says that a health check is a message: namely, one of the health messages that a Ceph cluster can raise. But it might be that this page is at odds with common usage of the term and is in need of revision on this point.

Reply:
Ack, seems off to me, but not a hill I'll die on.